### PR TITLE
#11 - Update slice added

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #11 
Added update slice and trivial test

### CODE REVIEW

File: src/main/java/com/artipie/debian/http/DebianUpdateSlice.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+import java.util.concurrent.CompletionStage;
+import java.util.function.Function;
+import org.reactivestreams.Publisher;
+
+/**
+ * Debian update slice adds uploaded slice to the storage and updates Packages index.
+ * @since 0.1
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
+ */
+public final class DebianUpdateSlice implements Slice {
```
Comment (olegmoz) [inline src/main/java/com/artipie/debian/http/DebianUpdateSlice.java:54]: @olenagerasimova I'd suggest to remove `Debian` part from the `Slice` name, as it is not likely to be used from outside of adapter code directly and in context of adapter code specifying that it's Debian is not needed

**Review by olegmoz — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, please see one comment

**Comment by olenagerasimova** [comment]
@olegmoz thanks, corrected

**Review by olegmoz — APPROVED** [review]
@olenagerasimova thanks, approved

**Pull request merged**